src/components/MovieInfo.js

```diff
@@ -1,26 +1,44 @@
 import React, { Component } from 'react';
+import apiCalls from '../apiCalls';
 
-function MovieInfo({movie}) {
+class MovieInfo extends Component {
+    constructor() {
+      super()
+      this.state = {
+        movie: null
+      }
+    }
+
+    componentDidMount = () => {
+      apiCalls.fetchAMovie(this.props.id)
+      .then(data => this.setState({movie: data.movie}))
+    }
+
+    render() {
+    if(!this.state.movie){
```

Comment:
How could this be written in a way that uses the conventional conditional rendering syntax?